recipe/meta.yaml

```diff
@@ -21,41 +23,49 @@ requirements:
     - python                              # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - cython                              # [build_platform != target_platform]
-    - numpy                               # [build_platform != target_platform]
+    - numpy >=2.0.0                       # [build_platform != target_platform]
```

Comment:
Please leave the numpy version in the build and host section blank. 

Conda forge manages build time pinnings in a global way

Reply:
Fixed in 1018cf3.  Thanks!